File: 37-Java/src/Prenotazione.java
public class Prenotazione {
    protected int codice;
    protected int oraInizio;
    
    public Prenotazione(int codice, int oraInizio) {
        this.codice = codice;
        this.oraInizio = oraInizio;
    }

    public String toString(){
        return "("+ codice +","+ oraInizio +")";
    }
}

File: 37-Python/Prenotazione.py
class Prenotazione:
    def __init__(self, codice, oraInizio) -> None:
        self._codice = codice
        self._oraInizio = oraInizio
    
    def __str__(self) -> str:
        return "("+ str(self._codice) +","+ str(self._oraInizio) +")"